client/components/date-range/header.tsx

```diff
@@ -22,10 +22,21 @@ const DateRangeHeader: FunctionComponent< Props > = ( {
 
 	return (
 		<div className="date-range__popover-header">
-			<Button className="date-range__cancel-btn" onClick={ onCancelClick } compact>
+			<Button
+				className="date-range__cancel-btn"
+				onClick={ onCancelClick }
+				compact
+				aria-label="Cancel"
+			>
 				{ cancelButtonText || translate( 'Cancel' ) }
 			</Button>
-			<Button className="date-range__apply-btn" onClick={ onApplyClick } primary compact>
+			<Button
+				className="date-range__apply-btn"
+				onClick={ onApplyClick }
+				primary
+				compact
+				aria-label="Apply"
```

Comment:
Should we localize this string as well?

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/63879/commits/9456a33036915934b4f62bf0086b2b4d372ce8c0.